src/MobileDevice/widget/MobileDevice.ts

```diff
@@ -124,6 +137,7 @@ class MobileDevice extends WidgetBase {
 
     private executeDeviceAction(mxObject: mendix.lib.MxObject) {
```

Comment:
Issue described in PR number 3, The getAppInformation this.executeDeviceAction(this.mxObject); even if there is no attribute configured.

Reply:
Issue confirmed